docs/tutorial/t-create-client-credentials.md

```diff
@@ -0,0 +1,399 @@
+# Create Client Credentials
+
+After having succesfully deployed a Kafka cluster you most probably would like to have client applications to connect to this cluster.
+
+## Install required software
+
+### Keytool (Java)
+
+If your clients applications are based on Java, consider that it might need to use special Java file formats for the certificates called keystore and trustore.
```

Comment:
```suggestion
If your client applications are based on Java, consider that it might need to use special Java file formats for the certificates called keystore and truststore.
```